rules.ini

```
@@ -51,6 +51,7 @@ Clausewitz[] = ^(?:game/)?common/defines/00_defines\.txt$
 ClickTeamFusion = (?:^|/)mmfs2\.dll$
 Cocos = cocos(?:2d|-js)
 Construct = (?:^|/)c[23]runtime\.js$
+CopperCube = \.cc(?:b|p)$
```

```suggestion
CopperCube = \.ccb$
```
removes a false positive https://steamdb.info/app/38600/depots/